setup.py

```diff
@@ -11,7 +11,7 @@
     author = 'Issam Zeibak',
     author_email = 'issam.zeibak@equityzen.com',
     url = 'https://github.com/equityzen/modern-treasury',
-    download_url = 'https://github.com/EquityZen/modern_treasury/archive/refs/tags/v_0.0.17.tar.gz',
+    download_url = 'https://github.com/EquityZen/modern_treasury/archive/refs/tags/v_0.0.18.tar.gz',
```

Comment:
@yazan-equityzen if you look a bit above this file:

`version = '0.0.17',` this shoudl also be bumped to 0.0.18